### PR TITLE
feat: add todo app blogpost / written tutorial

### PR DESCRIPTION
**Pull request by kaleem68**
#### Checklist

- [ ] run `make test`
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Code of conduct](https://github.com/wundergraph/wundergraph/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: docs-website/config/navigation.js [hunk 1]
```diff
@@ -156,10 +156,10 @@ const navigation = [
 				title: 'Your first WunderGraph Application',
 				href: '/docs/tutorials/your-first-wundergraph-application',
 			},
-			/*{
-				title: 'NextJS Full-Stack Tutorial',
-				href: '/docs/tutorials/nextjs-full-stack-tutorial',
-			},*/
+			{
+				title: 'Nextjs Todos Tutorial',
```
Comment (Pagebakers) [inline docs-website/config/navigation.js:160]: Build a Next.js Todo application

File: docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,1188 @@
...
+This will create the app in the `nextjs-todos` directory once it finishes, cd into it.
+
+```typescript
+cd nextjs-todos
+```
+
+Install dependencies and configure tailwindcss.
+
+```typescript
+npm install @heroicons/react @tailwindcss/forms clsx framer-motion
```
Comment (Pagebakers) [inline docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md:35]: I think some tailwind dependencies are missing here?

File: docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,1188 @@
...
+  order     Int     @default(0)
+}
+```
+
+The `order` field in Todo decides the reordering of todos. We will discuss later in the tutorial how to do reordering.
+
+To generate types and code for Todo, close the app, remove the cache and restart.
+
+- Close the app
+- Remove .wundergraph/cache
```
Comment (Pagebakers) [inline docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md:149]: Normally this shouldn't be needed

File: docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,1188 @@
...
+import { Fragment, useState } from 'react'
+
+import { Todo } from '../types'
+
+interface TodoItemProps {
+  todo: Todo
+}
+
+function TodoItem(props: TodoItemProps) {
+  console.log(props)
```
Comment (Pagebakers) [inline docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md:226]: What's this doing here

**Review by Pagebakers — CHANGES_REQUESTED** [review]

File: docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md (hunk 3 above)
Comment (kaleem68) [reply docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md:149]: When modifying `schema.prisma` it seems like it doesn't regenerate unless I remove `.wundergraph/cache`. If you want, I can skip the whole points to let the user decide how to regenerate.

File: docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md (hunk 4 above)
Comment (kaleem68) [reply docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md]: Removed.

File: docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md (hunk 3 above)
Comment (kaleem68) [reply docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md]: Removed the steps.

File: docs-website/config/navigation.js (hunk 1 above)
Comment (kaleem68) [reply docs-website/config/navigation.js]: Renamed to Build a Next.js Todo application.

File: docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md (hunk 2 above)
Comment (kaleem68) [reply docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md]: I have added those. Check.

File: docs-website/src/pages/docs/tutorials/build-a-next.js-todo-application.md [hunk 5]
```diff
@@ -0,0 +1,1174 @@
...
+```
+
+```css
+/* styles/globals.css */
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+```
+
```
[the remark this answers is not in the document]
Comment (kaleem68) [reply docs-website/src/pages/docs/tutorials/build-a-next.js-todo-application.md:68]: These warnings are not specific to the code; they exist everywhere in the docs, moving on, for now, I will create a proper issue for these warnings.

File: docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md (hunk 3 above)
Comment (kaleem68) [reply docs-website/src/pages/docs/tutorials/nextjs-todos-tutorial.md]: Check now updated the command.

**Pull request closed without merging**